content/en/storage-providers/advanced-configurations/workers.md

```diff
@@ -0,0 +1,685 @@
+---
+title: "Workers"
+description: "Lotus has a lot of advanced configurations you can tune to optimize your storage provider setup. This guide explains the advanced configuration options for Seal and PoSt workers"
+lead: "Lotus has a lot of advanced configurations you can tune to optimize your storage provider setup. This guide explains the advanced configuration options for Seal and PoSt workers"
+draft: false
+menu:
+    storage-providers:
+        parent: "storage-providers-advanced-configurations"
+weight: 110
+toc: true
+---
+
+While the Lotus workers have very reasonable default settings, some storage providers might want to fine tune some advanced configurations according to their setup.
+
+## Resource allocation in Seal Workers
+
+Each **Seal Worker** can potentially run multiple tasks in available slots. Each slot is called a _window_. The number of available windows per worker is determined by the requirements of the sealing tasks being allocated to the worker and its available system resources, including:
+
+- Number of CPU threads the task will use.
+- Minimum amount of RAM required for good performance.
+- Maximum amount of RAM required to run the task, where the system can swap out part of the RAM to disk, and the performance won't be affected too much.
+- Whether the system has a GPU.
+
+### Task resource table
+
+The default resource table lives in [resources.go](https://github.com/filecoin-project/lotus/blob/master/extern/sector-storage/storiface/resources.go) and can be edited to tune the scheduled behavior to fit specific sealing clusters better.
+
+Default resource value table. Some of these values are _fairly_ conservative:
```

Comment:
```suggestion
Here is the default resource value table. There values are conservative:
```

What does _conservative_ mean in this context: are we inflating these values, or deflating them?